.github/workflows/linting.yml

```diff
@@ -11,4 +11,4 @@ jobs:
       - name: Lint Bash
         uses: azohra/shell-linter@latest
         with:
-          path: '*.sh'
+          path: 'cleanup.sh,deploy.sh'
```

Comment:
I'm confused with this change. The issue seems to indicate that these files weren't being recognized because of the shebang line. Why do we need to list them specifically if those lines are being corrected?

Reply:
You are right, I can go back to the wildcard.

Reply:
resolved